fava/static/sass/_base.scss

```diff
@@ -595,3 +630,36 @@ table.queryresults {
         margin-left: 3px;
     }
 }
+
+/* Modifications for ipad */
```

Comment:
Maybe explain what's different to phones and why in this comment. Won't this also be active on some of these huge modern phones too?